lace/emulator/gp_emulator.py

```diff
@@ -51,6 +52,7 @@ def __init__(self,basedir="/lace/emulator/sim_suites/Australia20/",
         self.reduce_var_k=reduce_var_k ## Emulate (1+k)P1D(k)
         self.reduce_var_z=reduce_var_z ## Emulate P1D(k)/(1+z)^3.8
         self.reduce_var_mf=reduce_var_mf ## Emulate P1D(k)*<F>^2.5
+        self.key_data=key_data
```

Comment:
It would be great to add a comment explaining what key_data is, and how it should be used

Reply:
I have included an clarification of what key_data selects. In the case of the GP, the key_data is hardcoded, it should not be used. Should I add anything else here?

Reply:
You have probably added this clarification in the other branch, since I don't see any change. Given that this branch should focus only on the changes in the arXiv, and that this `key_data` is not needed for that, I would like to propose to go back to the previous version of this file. We can add back `key_data` when we merge your NN branch.